x-pack/plugins/security_solution/package.json

```diff
@@ -16,6 +16,8 @@
     "cypress:dw:serverless": "NODE_OPTIONS=--openssl-legacy-provider node ./scripts/start_cypress_parallel --config-file ./public/management/cypress/cypress_serverless.config.ts --ftr-config-file ../../test/defend_workflows_cypress/serverless_config",
     "cypress:dw:serverless:open": "yarn cypress:dw:serverless open",
     "cypress:dw:serverless:run": "yarn cypress:dw:serverless run",
+    "cypress:dw:qa:serverless": "NODE_OPTIONS=--openssl-legacy-provider node ./scripts/start_cypress_parallel_serverless --config-file ./public/management/cypress/cypress_serverless_qa.config.ts",
```

Comment:
Suggestion: for clarity - maybe instead of `:qa:` use `:mki:`?

Reply:
_coming back here (after completing code review)_: I see that `qa` was used through out. I'm fine with just leaving it as is.